### DISCUSSION TITLE
Safari reloading issue if checkLoginIframe is enabled

### DISCUSSION THREAD

**Discussion by MatejFacko** [keycloak/keycloak#19825] [Q&A] [answered]
Good Morning,

We are using KeycloakJS adapter (v. 19.0.2).  On the page load we are initialising the Keycloak client with the init function.

`keycloak.init({
    onLoad: "check-sso",
    pkceMethod: "S256",
    silentCheckSsoFallback: false,
    checkLoginIframe: true,
})`

On Safari we are experiencing the reloading issue. In the network tab we can see requests step1.html and step2.html are failling see attached screenshot bellow. 

![Screenshot 2023-04-20 at 07 11 14](https://user-images.githubusercontent.com/76104834/233265373-183f838e-1945-471f-aa32-06d8c6a8d86e.png)

![Screenshot 2023-04-20 at 07 11 18](https://user-images.githubusercontent.com/76104834/233265389-82f34b46-52aa-4141-ae02-0db2c58e3a8c.png)

For every other browser there is no issue with the step1.html and step2.html. (Checked in Chrome, Firefox, Opera, Edge)

**Comment by jonkoops** [comment]
This bug has been fixed in newer versions of Keycloak I believe. Try installing the latest version of Keycloak.

**Reply by MatejFacko** [reply]
> This bug has been fixed in newer versions of Keycloak I believe. Try installing the latest version of Keycloak.

@jonkoops  I switch to    "keycloak-js": "21.0.2", but the problem is still the same

**Reply by jonkoops** [reply, marked as answer]
The fix is in the Keycloak server itself as well, you'll have to update both the server and your client version.

**Reply by MatejFacko** [reply]
@jonkoops  you're right, confirmed it's working. thank you.
